docs/content/latest/releases/whats-new/stable-releases.md

```diff
@@ -21,6 +20,85 @@ Included here are the release notes for all releases in the v2.4 stable release
 
 {{< /note >}}
 
+## v2.4.2.0 - April 26, 2021
+
+### Downloads
+
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.4.2.0-darwin.tar.gz">
+  <button>
+    <i class="fab fa-apple"></i><span class="download-text">macOS</span>
+  </button>
+</a>
+&nbsp; &nbsp; &nbsp;
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.4.2.0-linux.tar.gz">
+  <button>
+    <i class="fab fa-linux"></i><span class="download-text">Linux</span>
+  </button>
+</a>
+<br />
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.4.2.0-b42
+```
+
+### Uncategorized issues
+
+#### Core Database
+
+* [BACKPORT 2.4] Update sbt version to 0.13.18
+* [BACKPORT 2.4] [[5383](https://github.com/yugabyte/yugabyte-db/issues/5383)] Fix conflict resolution while adding read intents
```

Comment:
improvements
* [BACKPORT 2.4] [[6241](https://github.com/yugabyte/yugabyte-db/issues/6241)] YSQL: Make row key share lock less restrictive
* [BACKPORT 2.4] [[6305](https://github.com/yugabyte/yugabyte-db/issues/6305)] Adaptive Heartbeat Reporting
* [BACKPORT 2.4] [[6445](https://github.com/yugabyte/yugabyte-db/issues/6445)] docdb: Master should rebuild YCQL system.partitions on a background thread
* [BACKPORT 2.4] [[7355](https://github.com/yugabyte/yugabyte-db/issues/7355)] ysql: check backfill bad connection status
* [BACKPORT 2.4] [[7705](https://github.com/yugabyte/yugabyte-db/issues/7705)] ysql: prioritize internal HBA config
* [Backport 2.4] [[5647](https://github.com/yugabyte/yugabyte-db/issues/5647)] docdb: LB should not move tablets right after splitting
* [Backport 2.4] [[7079](https://github.com/yugabyte/yugabyte-db/issues/7079)] [[7153](https://github.com/yugabyte/yugabyte-db/issues/7153)] YSQL: Improve configuration and defaults for client to server encryption
* [Backport 2.4] [[7455](https://github.com/yugabyte/yugabyte-db/issues/7455)] [YCQL] Update index from transaction with cross-key statements.
* [Backport 2.4] [[7641](https://github.com/yugabyte/yugabyte-db/issues/7641)] YCQL: Fix checks in index update path that determine full row removal.
* [Backport 2.4] [[7813](https://github.com/yugabyte/yugabyte-db/issues/7813)] [YSQL] YSQL dump should always include HASH/ASC/DESC modifier for indexes/pkey.


bug fixes
* [BACKPORT 2.4] [[5383](https://github.com/yugabyte/yugabyte-db/issues/5383)] Fix conflict resolution while adding read intents
* [BACKPORT 2.4] [[6747](https://github.com/yugabyte/yugabyte-db/issues/6747)] [[6902](https://github.com/yugabyte/yugabyte-db/issues/6902)] [[7109](https://github.com/yugabyte/yugabyte-db/issues/7109)] [7284] YCQL system.partition bug fixes
* [BACKPORT 2.4] [[6960](https://github.com/yugabyte/yugabyte-db/issues/6960)] docdb - Register ScopedRWOperation when accessing doc_db data in StillHasParentDataAfterSplit
* [BACKPORT 2.4] [[7105](https://github.com/yugabyte/yugabyte-db/issues/7105)] Explicitly clear the global PgMemctx map in YBCDestroyPgGate
* [BACKPORT 2.4] [[7145](https://github.com/yugabyte/yugabyte-db/issues/7145)] Fix wrong propagated safe time
* [BACKPORT 2.4] [[7397](https://github.com/yugabyte/yugabyte-db/issues/7397)] Tablet splitting: Crash during generating TSHeartbeat report after split
* [BACKPORT 2.4] [[7835](https://github.com/yugabyte/yugabyte-db/issues/7835)] Don't crash when trying to append ValueType::kTombstone to a key



can be ignored -- test or release only
* [BACKPORT 2.4] [[7289](https://github.com/yugabyte/yugabyte-db/issues/7289)] Test split in progress should not block LB data move if blacklist is set
* [Backport 2.4] Update NFS shared path /n to /Volumes/n
 * [Backport 2.4] [[7054](https://github.com/yugabyte/yugabyte-db/issues/7054)] Fix undefined variable in ListTables conditional check.
* [Backport 2.4] [[7628](https://github.com/yugabyte/yugabyte-db/issues/7628)] Add ldap libraries as special case for yb client packaging

platform
* [Backport 2.4] [[7769](https://github.com/yugabyte/yugabyte-db/issues/7769)] Prevent adding on-prem node instance with duplicate IP